### ISSUE TITLE
Issue in executing the demo

### ISSUE THREAD

**Issue by agarwalayush** [isislab/dispatch#2]
Hi,
I compiled `thing.c` in `demos` with gcc on linux, and when I run the `patch.py`, I get the following weird error:
```
 File "patch.py", line 16, in <module>
    exe.replace_instruction(ins, '')
  File "build/bdist.linux-x86_64/egg/dispatch/formats/elf_executable.py", line 292, in replace_instruction
  File "build/bdist.linux-x86_64/egg/dispatch/util/trie.py", line 62, in __contains__
TypeError: unsupported operand type(s) for >>: 'Instruction' and 'int'
```

Can you please help me with this?

**Comment by computerfreak** [comment]
Hi Ayush,

Thanks for the report. I believe this is due to me not having completely switched over the implementations for `replace_instruction` (we changed what the method takes to let you replace a sequence of instructions instead of just one). I'll fix this up later today/tomorrow.

**Comment by computerfreak** [comment]
Fixed in cfaa93f
